toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -346,49 +346,86 @@ Specifies the configuration for the generated ISO media.
 
 Specifies the configuration for overlay filesystem.
 
-- `lowerDir`: This directory acts as the read-only layer in the overlay
-  filesystem. It contains the base files and directories which will be overlaid
-  by the upperDir. Changes to the overlay filesystem do not affect the contents
-  of lowerDir.
+- `lowerDir`: (Required)
 
-- `upperDir`: This directory is the writable layer of the overlay filesystem.
-  Any modifications, such as file additions, deletions, or changes, are made in
-  the upperDir. These changes are what make the overlay filesystem appear
-  different from the lowerDir alone.
+  This directory acts as the read-only layer in the overlay filesystem. It
+  contains the base files and directories which will be overlaid by the
+  upperDir. Changes to the overlay filesystem do not affect the contents of
+  lowerDir. 
+  
+  Example: `/etc`
 
-- `workDir`: This is a required directory used for preparing files before they
-  are merged into the upperDir. It needs to be on the same filesystem as the
-  upperDir and is used for temporary storage by the overlay filesystem to ensure
-  atomic operations. The workDir is not directly accessible to users.
+- `upperDir`: (Required)
 
-- `partition`: Optional field: If configured, a partition will be attached to
-  the current targeted overlay, making it persistent and ensuring that changes
-  are retained. If not configured, the overlay will be volatile.
+  This directory is the writable layer of the overlay filesystem. Any
+  modifications, such as file additions, deletions, or changes, are made in the
+  upperDir. These changes are what make the overlay filesystem appear different
+  from the lowerDir alone. 
+  
+  Example: `/var/overlays/etc/upper`
 
-  - `idType`: Specifies the type of id for the partition. The options are
-    `part-label` (partition label), `uuid` (filesystem UUID), and `part-uuid`
-    (partition UUID).
+- `workDir`: (Required)
 
-  - `id`: The unique identifier value of the partition, corresponding to the
-    specified IdType.
+  This is a required directory used for preparing files before they are merged
+  into the upperDir. It needs to be on the same filesystem as the upperDir and
+  is used for temporary storage by the overlay filesystem to ensure atomic
+  operations. The workDir is not directly accessible to users. 
+  
+  Example: `/var/overlays/etc/work`
+
+- `mountPoint`: (Required)
+
+  The directory where the overlay filesystem will be mounted.
```

Comment:
nit: maybe indicate that this is where the combined view of upper/lower will be mounted.

Reply:
thx!